tool/password/src/main/java/com/github/mizool/tool/password/PasswordHashTool.java

```diff
@@ -39,12 +43,16 @@ public static void main(String[] args)
             }
             else
             {
-                PasswordHashTool passwordHashTool = new PasswordHashTool(parameters.getPassword());
-                String digest = passwordHashTool.getDigest();
-                if (digest != null)
-                {
-                    System.out.println("Digest: " + digest);
-                }
+                Iterable<PasswordHasher> instances = MetaInfServices.instances(PasswordHasher.class);
+                instances.forEach(hasher -> {
```

Comment:
While we're cleaning up things: there's no reason to instantiate the `PasswordHashTool` - its instance methods only consist of one-liners that could be equally placed at the call site. Therefore, drop fields, constructor and instance methods.

Still, having some separation between high-level code in `main` and other stuff is useful: let's replace this lambda with a reference to a new static method `void printDigest(PasswordHasher hasher)`.